stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -148,6 +148,14 @@ public void setInspectModeEnabled(boolean enabled) {
     }
   }
 
+  @Override
+  public void setAttributesAsText(Object element, String text) {
```

Comment:
The first thing this method needs to do is call `verifyThreadAccess()`. We want to do that at the start of any method that is potentially dangerous if misused (e.g. `hook()` and `unhook()`), and also for any method that writes to an element (which this one definitely does). See the 4th paragraph of the long-winded javadocs for `ChainedDescriptor` for my explanation of this.